CONTRIBUTING.md

```diff
@@ -66,14 +64,23 @@ export USERNAME={userName}
 export PASSWORD={password}
 ```
 
-After setting up the environment variables, run this script to copy the configuration into the JSON configuration files:
+For Windows, please set the following environment variables - ISSUER, CLIENT_ID, USER_NAME, PASSWORD
+
+> **NOTE:** Windows has USERNAME as a built-in system variable, hence set the USER_NAME environment variable for testing.
+
+After setting up the environment variables, run this node script to copy the configuration into the JSON configuration files:
 
-```bash
-sh scripts/setup-env.sh
+```
+node scripts/setup-env.js
 ```
 
-Then run the E2E tests:
+If you're on a *nix based machine, run the E2E tests:
 
 ```bash
 npm test
 ```
+
+If you're on Windows, run the following command:
+```bash
+npm run test-windows
```

Comment:
Is it necessary to have a separate command? It looks like this just triggers a bash script, which should work in either environment (also, we're assuming that Windows users are using something bash-like?)

Reply:
Copy pasting a comment made before on why it's needed - 
Need this script instead of simply running `npm run test:okta-hosted-login && npm run test:custom-login` because - 
* I've added a "hack" in TCK that will kill the `node.exe` process after any `npm` script exits. E.g - `npm run test:okta-hosted-login` will start `npm run okta-hosted-login-server`. After the 1st script exits, I kill all `node.exe` processes which in turn kills the 1st script also
* Due to this "hack" (found no way around in windows to just kill the child process), only the 1st npm script in `npm run test:okta-hosted-login && npm run test:custom-login` will run.
* To run both the tests/scripts, I have to run them one after another so that no unwarranted killing happens 
* I know it's messy but I spent enough time to find a better way but couldn't come up with a better solution

Reply:
Also this script was able to run on windows powershell, which I used for testing.
That makes me believe that I should also test it on command prompt.
I can convert this to a batch file if needed.